includes/notes/class-wc-admin-note.php

```diff
@@ -347,6 +347,12 @@ public function set_content( $content ) {
 			'br'     => array(),
 			'em'     => array(),
 			'strong' => array(),
+			'a'      => array(
+				'href'   => true,
+				'rel'    => true,
+				'name'   => true,
+				'target' => true,
```

Comment:
Do you think we could add the `download` attribute here? I can imagine some legit uses, and WordPress also allows it here: https://core.trac.wordpress.org/browser/trunk/src/wp-includes/kses.php#L63.

Reply:
Added in e27e31b.